AppCenter/AppCenter/Internals/AppDelegate/MSAppDelegateForwarder.m

```diff
@@ -6,21 +6,11 @@
 #import "MSLogger.h"
 #import "MSUtility+Application.h"
 
-// Enum used to represent all kind of executors running the completion handler.
```

Comment:
Can't we keep this here? Seems this can be used by any other delegates with completionHandler.

Reply:
I agree.

Reply:
We can so that

Reply:
I addressed it.